### PR TITLE
LOM-266 Profile block

### PR DESCRIPTION
**Pull request by teroelonen**
# [LOM-266](https://helsinkisolutionoffice.atlassian.net/browse/LOM-266)

## What was done

* Added a module called helfi_profile_block that adds Profile block functionality to the header.

## How to install

* Check installation instructions from the hdbt PR linked in this PR.

## How to test

* [x] Check testing instructions from the hdbt PR linked in this PR.
* [ ] Check that code follows our standards

## Other PRs

* https://github.com/City-of-Helsinki/drupal-hdbt/pull/523

[LOM-266]: https://helsinkisolutionoffice.atlassian.net/browse/LOM-266?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: helfi_features/helfi_profile_block/templates/profile-block.html.twig [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% if logged_in %}
```
Comment (Arkkimaagi) [inline helfi_features/helfi_profile_block/templates/profile-block.html.twig:1]: This file should be in HDBT, not in platform-config

File: helfi_features/helfi_profile_block/templates/profile-block.html.twig [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  {% set dropdown_id = 'profile-dropdown' %}
+
+  {% embed "@hdbt/misc/nav-toggle-button.twig" with {
+    modifier_class: 'nav-toggle--profile',
+    controls: dropdown_id,
+    anchor_target: '#profile',
+    js_target: 'js-profile-button',
+
+    open_label: display_name,
+    open_screenreader: 'Open profile information menu'|t({}, {'context': 'Profile information-menu open button text for screen readers'}),
```
Comment (Arkkimaagi) [inline helfi_features/helfi_profile_block/templates/profile-block.html.twig:17]: Should we contain some information here to the end screen reader end user that they're logged in as `display_name` like sighted users get?

**Review by Arkkimaagi — CHANGES_REQUESTED** [review]
A couple of notes, but seems to work nicely otherwise.

File: helfi_features/helfi_profile_block/templates/profile-block.html.twig (hunk 1 above)
Comment (teroelonen) [reply helfi_features/helfi_profile_block/templates/profile-block.html.twig]: Good catch this one! Forgot all about it. Moved it to hdbt.

File: helfi_features/helfi_profile_block/templates/profile-block.html.twig (hunk 2 above)
Comment (teroelonen) [reply helfi_features/helfi_profile_block/templates/profile-block.html.twig]: I added a text that tells screen reader user this information to the open_screenreader value.

**Review by Arkkimaagi — APPROVED** [review]
Seems to be working as it should, good job! 🌞

**Pull request merged**